tests/test_parser.py

```diff
@@ -150,3 +152,54 @@ def test_excel_service_json_generation(xls_input, expected_json_output):
     assert not results.synchronizations.missing
     assert not results.synchronizations.extra
     assert not results.synchronizations.different
+
+# test xls answers creation
+@pytest.mark.parametrize('json_input, csv_output', {
+    DATA_DIR / 'testTopology_response.json':     DATA_DIR / 'testTopology_response',
+}.items())
+def test_csv_response_generation(json_input, csv_output):
+    """ tests if generated csv is consistant with expected generation
+        same columns (order not important)
+    """
+    with open(json_input) as jsonfile:
+        json_data = load(jsonfile)
+    equipment = load_equipment(eqpt_filename)
+    csv_filename = str(csv_output)+'.csv'
+    with open(csv_filename, 'w', encoding='utf-8') as fcsv:
+        jsontocsv(json_data, equipment, fcsv)
+
+    expected_csv_filename = str(csv_output)+'_expected.csv'
+
+    # expected header
```

Comment:
yes csv_header variable not useful anymore, but its content may help for debugging so I preferred to leave it as an information/comment

Reply:
I do not understand. These data are present in the CSV file, and you said that you're using the panda library so that you can have per-column access to the data sets -- right? What's the point in copying that info in here? I'm not saying that I am right, I'm asking what I am missing.